csharp/nuget/Facebook.Yoga.Native.nuspec

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Facebook.Yoga.Native</id>
+    <version>3.0.0-pre</version>
+    <title>Facebook.Yoga.Native</title>
+    <authors>Facebook</authors>
+    <owners>Facebook</owners>
+    <licenseUrl>https://github.com/facebook/css-layout/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/facebook/css-layout</projectUrl>    
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>A subset of CSS's flexbox layout algorithm and box model.</description>
+    <copyright>Copyright 2016 Facebook</copyright>
+    <tags>native flexbox flex-box css layout css-layout yoga facebook</tags>
+    <dependencies>
+      <group targetFramework=".NETStandard1.0" />
+    </dependencies>
+  </metadata>
+
+  <files>
+    <!-- build -->
+    <file src="Facebook.Yoga.Native.targets" target="build\netstandard1.0"/>
+
+    <!-- lib -->
+    <file src="_._" target="lib\netstandard1.0"/>
```

Comment:
@rozele It sort of is magic and it seems hard to find definitive documentation on this technique... I originally saw it in sqlite.native and many other libraries that referenced sqlite related packages... Libraries like Microsoft.BCL also use this technique (see its win8 folder).... 

A search for _._ in the NuGet gitHub reveals the constant "PackageEmptyFileName" and that constant is used in the logic of NuGet packager/client...

From what I can gather the NuGet client needs to have some lib\targetframeworkmoniker (TfxM) in order for it to make the package reference-able from another .NET project.  Since this nuspec represents a package with only the native DLL we don't actually have a managed library to place into the folder.  Nuget automatically strips out empty folders so we cannot leave an empty TfxM folder... To avoid NuGet from excluding the empty folder we put an _._ file to create a non-empty folder.   And to avoid copying the useless _._ file to the the NuGet source ignores it... 

There is a link [here](https://github.com/fsprojects/Paket/issues/193)  that sort of discusses it...